latticexyz/mud
# chore: add release:yalc script for local release testing

Steps to test a release locally:
1. Run `yarn release:yalc` in mud root
2. Run `npx yalc add @latticexyz/<packageToTest>` in root of the project you want to test with
3. Verify things work
4. Run `npx yalc remove @latticexyz/<packageToTest> && yarn --force` in test project to reset it to the normal state (without yalc package)
